cSpell.json

```diff
@@ -79,5 +81,7 @@
         "Frontmatter",
         "Liquid Includes"
     ],
-    "ignorePaths": []
+    "ignorePaths": [
+        "docs/_pages/releases.md"
+    ]
```

Comment:
I think this should be reverted, because on develop it is empty and no spell error is thrown.

Note, that I have added a few more ignore patterns back than

Reply:
For reference: https://github.com/fluentassertions/fluentassertions/pull/2361